src/api/routes/auth.py

```
@@ -0,0 +1,256 @@
+        'Password was reset successfully',
+        request.remote_addr
+    )
+    
+    return jsonify({'message': 'Password reset successful'}), 200
+
+@auth_bp.route('/audit-logs', methods=['GET'])
+@jwt_required()
+def get_audit_logs():
+    """Get audit logs for the current user."""
+    user_id = get_jwt_identity()
+    user = User.query.get(user_id)
```

Ruff (0.11.9)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)